otherFiles/Person.java

```diff
@@ -0,0 +1,64 @@
+package other;
+
+public final class Person {
+    private final int id;
+    private final int phone;
+    private final int additionalPhone;
+    private final int officeNumber;
+    private final int luckyNumber;
+
+    private Person(Builder builder) {
+        id = builder.id;
+        phone = builder.phone;
+        additionalPhone = builder.additionalPhone;
+        officeNumber = builder.officeNumber;
+        luckyNumber = builder.luckyNumber;
+    }
+
+    public static class Builder {
+        private int id;
+        private int phone;
+        private int additionalPhone;
+        private int officeNumber;
+        private int luckyNumber;
+        public Builder setId(int i) {
```

Comment:
between methods and fields should be an empty line and between methods too. Change everywhere.